test/parallel/test-zlib.js

```diff
@@ -230,3 +230,12 @@ testKeys.forEach(common.mustCall((file) => {
     }, trickle.length));
   }, chunkSize.length));
 }, testKeys.length));
+
+{
+  // Test instantiation without 'new'
+  const gzip = zlib.Gzip();
```

Comment:
Nit, if you use the `--no-warnings` flag it will suppress the github actions reported warnings below but still keep `expectWarning(...)` working.